test/operationpoint/PowerModelsIntegration.jl

```diff
@@ -0,0 +1,89 @@
+using PowerDynamics: SlackAlgebraic, SwingEqLVS, PQAlgebraic, StaticLine, PowerGrid, find_operationpoint, VSIMinimal, power_flow
+using OrderedCollections: OrderedDict
+using Test: @test, @testset, @test_throws, @test_logs, @test_nowarn
+
+node_dict = OrderedDict(
+    "bus1" => SlackAlgebraic(U = 1.),
+    "bus2" => SwingEqLVS(H = 5., P = 1., D = 0.1, Ω = 50, Γ = 0.1, V = 1.),
+    "bus3" => SwingEqLVS(H = 5., P = 1., D = 0.1, Ω = 50, Γ = 0.1, V = 1.),# VSIMinimal(τ_P=1., τ_Q=2., K_P=3., K_Q=4., V_r=1, P=1., Q=0.),
+    "bus4" => PQAlgebraic(P = -1, Q = -1)) # note - for PowerModels it is different default direction, multiply by -1
+
+line_dict = OrderedDict(
+    "line1" => StaticLine(from = "bus1", to = "bus2", Y = -1im / 0.02),
+    "line2" => StaticLine(from = "bus2", to = "bus3", Y = -1im / 0.02),
+    "line3" => StaticLine(from = "bus3", to = "bus4", Y = -1im / 0.02))
+
+node_list = collect(values(node_dict))
+
+line_list = []
+push!(line_list, StaticLine(from = 1, to = 2, Y = -1im / 0.02))
+push!(line_list, StaticLine(from = 2, to = 3, Y = -1im / 0.02))
+push!(line_list, StaticLine(from = 3, to = 4, Y = -1im / 0.02))
+
+powergrid_dict = PowerGrid(node_dict, line_dict)
+powergrid = PowerGrid(node_list, line_list)
+
+##
+
+operationpoint = find_operationpoint(powergrid_dict; sol_method=:rootfind)
+
+data, result = power_flow(powergrid)
+
+# The problem is in this line.
+data_2, result_2 = power_flow(powergrid_dict)
+
+@test all(keys(data_2) .== keys(data))
+
+# compare bus data
+for (k, v) in data["bus"]
+    @test all(keys(v) .== keys(data_2["bus"][k]))
+    @test all(values(v) .== values(data_2["bus"][k]))
+end
+
+for (k, v) in data["gen"]
+    @test all(keys(v) .== keys(data_2["gen"][k]))
+    @test all(values(v) .== values(data_2["gen"][k]))
+end
+
+for (k, v) in data["load"]
+    @test all(keys(v) .== keys(data_2["load"][k]))
+    @test all(values(v) .== values(data_2["load"][k]))
+end
+
+# compare branch data
+for (k, v) in data["branch"]
+    @test all(keys(v) .== keys(data_2["branch"][k]))
+    @test all(values(v) .== values(data_2["branch"][k]))
```

Comment:
@SabineAuer the problem seems to be, that for the dict, the PowerModels data is not correctly created. As far as I can see, the `from`and `to` field are strings and not numbers, apparently this does not work with PM since there does not seem to be any other difference between `data`and `data_2`.